src/ResourceManager/Network/Commands.Network/Models/PSApplicationGatewayBackendHttpSettings.cs

```diff
@@ -30,6 +31,7 @@ public class PSApplicationGatewayBackendHttpSettings : PSChildResource
         public string HostName { get; set; }
         public bool? PickHostNameFromBackendAddress { get; set; }
         public string AffinityCookieName { get; set; }
+        [Obsolete("This property is ignored and will be removed in the future.")]
```

Comment:
@jobatzil please change this message to "Property ProbeEnabled is ignored and will be removed from the PSApplicationGatewayBackendHttpSettings type in a future release".